docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -365,271 +613,273 @@ is a dictionary which contains the following keys:
-See the :ref:`storm-cron` command in the :ref:`storm-ref-cmd` document for details.
-
-In :ref:`gloss-optic`, cron jobs can also be managed through the :ref:`gloss-admin-tool`.
-
-.. NOTE::
+.. TIP::
   
-  Once a cron job is created, you can modify many of its properties (such as its name and description, or
-  the Storm associated with the job). However, you cannot modify other aspects of the job, such as its
-  schedule. To change those conditions, you must delete and re-create the cron job.
+  For ``node:add`` and ``node:del`` triggers, the node that caused the trigger to fire is sent inbound
+  to the trigger's Storm.
+  
+  The ``maxmind`` and ``nettools.dns`` commands are added to Synapse by the `Synapse-Maxmind`_ and `Synapse-Nettools`_
+  Power-Ups, respectively.
 
 
-Examples
-~~~~~~~~
-
-In the examples below, we show the command to create (add) the specified cron job.
+**edge:add example**
 
-For illustrative purposes, in the **first** example the newly created cron job is then displayed using
-the ``cron.list`` command and by lifting the associated ``syn:cron`` runtime ("runt") node.
+You are using ``risk:attack`` nodes in Synapse to represent attacks, and linking the attack to key objects used in the attack
+with a ``-(uses)>`` light edge. (For example, a ``risk:attack`` representing a phishing attempt can be linked to the relevant
+email message (``file:bytes`` node) or email metadata nodes (such as ``inet:email:message``, ``inet:email:message:attachment``,
+or ``inet:email:message:link``).
 
-- Add a one-time / non-recurring cron job to run at 7:00 PM to create the RFC1918 IPv4 addresses in the
-  172.16.0.0/16 range.
+If the object "used" in the attack has an associated TTP tag (such as ``#cno.ttp.phish.attach`` to represent a malicious
+attachment used in a phishing attack), you want to pivot to the technique (``ou:technique``) represented by the tag (e.g., 
+"Spear phishing attachment"), and create another ``uses`` edge to show the attack uses the specified technique.
 
-.. storm-cli:: cron.at --hour 19 { [ inet:ipv4=172.16.0.0/16 ] }
+.. storm-cli:: trigger.add edge:add --name 'Link technique with attack'--verb uses --form risk:attack --query { $attack=$node.value() { -(uses)> * +$auto.opts.n2iden -> # -> ou:technique:tag [ <(uses)+ { risk:attack=$attack } ] } }
 
+.. TIP::
+  
+  Recall that edges have a direction, regardless of whether the edge relationship is specified "left to right" or
+  "right to left" in Storm (e.g., ``n1 -(uses)> n2`` or ``n2 <(uses)- n1``).
+  
+  For ``edge:add`` and ``edge:del`` triggers, the ``n1`` node (source node) for the edge that caused the
+  trigger to fire is sent inbound to the trigger's Storm.
 
-Newly created cron job via ``cron.list``:
+The Storm for this trigger is broken down below (with comments) for clarity:
 
-.. storm-cli:: cron.list
+::
+  
+  // Set the variable $attack to the guid of the risk:attack node for later use
+  $attack=$node.value()
```

Comment:
Rather than saving the node value here, you could save the entire node (`$attack=$node`) and yield it later rather than re-lifting it (`yield $attack`).

Reply:
clever